source/sass/mofo-bootstrap/_colors.scss

```diff
@@ -27,6 +27,7 @@ $pni-purple: #7f28b7;
 $pni-purple-pink: #b0379b;
 $pni-pink: #e4487d;
 $pni-gradient: linear-gradient(to right, $pni-blue 0%, $pni-pink 100%);
+$pni-warning-yellow: #fbd545;
```

Comment:
would it be worth making a `_colors__pni.scss` and importing that?

Reply:
Yup can do that. But will file a new ticket for it.

Reply:
Fileld https://github.com/mozilla/foundation.mozilla.org/issues/5315